miden-tx/tests/test_miden_faucet_contract.rs

```diff
@@ -207,6 +212,44 @@ fn test_faucet_contract_burn_fungible_asset_succeeds() {
     assert!(transaction_result.consumed_notes().notes()[0].hash() == note.hash());
 }
 
+#[test]
+fn test_faucet_contract_creation() {
+    // we need a Falcon Public Key to create the wallet account
+    let key_pair: KeyPair = rpo_falcon512::KeyPair::new().unwrap();
+    let pub_key: PublicKey = key_pair.public_key();
+    let auth_scheme: AuthScheme = AuthScheme::RpoFalcon512 {
+        pub_key: pub_key.into(),
+    };
+
+    // we need to use an initial seed to create the wallet account
+    let init_seed: [u8; 32] = [
+        90, 110, 209, 94, 84, 105, 250, 242, 223, 203, 216, 124, 22, 159, 14, 132, 215, 85, 183,
+        204, 149, 90, 166, 68, 100, 73, 106, 168, 125, 237, 138, 16,
+    ];
+
+    let faucet_meta_data: Word = [Felt::new(123), ZERO, Felt::new(456), Felt::new(789)];
+
+    let (faucet_account, _) =
+        create_basic_faucet(init_seed, faucet_meta_data, auth_scheme).unwrap();
```

Comment:
question: can't we use the `mock` for the new tests?

Reply:
I don't quite understand that comment. Can you show me what you mean? Move this test to `mock` or use the `mock` to create a basic faucet?